eZ/Publish/Core/Repository/Tests/Service/Mock/RoleTest.php

```diff
@@ -1139,12 +1139,14 @@ protected function getPartlyMockedRoleService(
      * @param string[] $methods
      * @param array $settings
      *
-     * @return \eZ\Publish\Core\Repository\Helper\LimitationService|\PHPUnit\Framework\MockObject\MockObject
+     * @return \eZ\Publish\Core\Repository\Permission\LimitationService|\PHPUnit\Framework\MockObject\MockObject
      */
     protected function getPartlyMockedLimitationService(array $methods = null, array $settings = [])
     {
         if (!isset($this->partlyMockedLimitationService) || !empty($settings)) {
-            $this->partlyMockedLimitationService = $this->getMockBuilder('eZ\\Publish\\Core\\Repository\\Helper\\LimitationService')
+            $this->partlyMockedLimitationService = $this->getMockBuilder(
+                'eZ\\Publish\\Core\\Repository\\Permission\\LimitationService'
+            )
```

Comment:
```suggestion
            $this->partlyMockedLimitationService = $this->getMockBuilder(LimitationService::class)
```

Reply:
Entirely gone via 5b1b20916c (or rather moved to BaseTest).